ftp/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -3,68 +3,35 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-FTP Connector v1.5
-
-Anypoint Connector for FTP (FTP Connector) provides access to files and folders on an FTP server. Its main features include:
+Anypoint Connector for FTP (FTP Connector) provides access to files and folders on an FTP server. FTP connector operations manage file transfers over the FTP (File Transfer Protocol) protocol. The connector's main features include:
 
 * The ability to read files or fully list directory contents on demand.
 * Support for common FTP operations such as creating directories and copying, moving, renaming, and deleting files.
 * Support for locking files.
 * File matching functionality.
-* A design that is fully consistent with the File and SFTP connectors. The same set of operations is available on all these connectors, and they behave similarly.
-
-
-== POM File Information
-
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>org.mule.connectors</groupId>
-  <artifactId>mule-ftp-connector</artifactId>
-  <version>RELEASE</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
-
-Mule converts RELEASE to the latest version. To specify a version, view
-https://www.mulesoft.com/exchange/org.mule.connectors/mule-ftp-connector/[FTP Connector] in
-Anypoint Exchange and click *Dependency Snippets*.
+* A design that is consistent with the File and FTP connectors.
++
+The same set of operations is available on these connectors, and they behave similarly.
 
-[[connection_settings]]
-== Connect to an FTP Server
+=== Representation of File Attributes
 
-You can set up a connection through an `ftp:config` element, for example:
+Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
 
-[source,xml,linenums]
-----
-<ftp:config name="ftp">
-  <ftp:connection
-    username="anonymous"
-    password="password"
-    host="localhost"
-    port="${ftpPort}"
-    workingDir="${workingDir}"/>
-</ftp:config>
-----
+The Mule message contains the concepts of encoding and MIME type. These describe the format in which a payload is represented. The file attributes describe a file’s encoding and MIME type.
```

Comment:
These sentences repeat "encoding and MIME type," which is probably unnecessarily wordy.
How about tacking this info onto the previous?
```suggestion
In Mule 4, messages comprise _file content_ (a payload) and _file attributes_ (metadata about the payload, such the file’s name, path, size, and timestamps. The file attributes also describe the format in which a payload is represented by including a file’s encoding and MIME type.
```